PowerShell Contributing.md

```diff
@@ -0,0 +1,399 @@
+
+The maintainers will look at your idea and let you know what they think. All genuine feature requests are handled with respect.
+So don't be afraid to share your thoughts! 
+
+Do take the time to find out if your idea has been proposed before and what was decided and why.
+You can filter on the label 'Issue-Enhancement'.
+
+
+In some cases it may be a good move to 'test' your idea against a few people first, before taking the time to fully explain what you have in mind in an issue, a great place to do so would be the [PowerShell Discord/Slack/IRC channel](link required).
+
+### 6. Contribute Help content
+
+> Requires
+> 
+> - GitHub account
+> - Knowledge of Markdown
+> 
+> Optional
+> 
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+
+You can contribute by writing, improving, expanding help files.
+The ones that are used by Get-Help.
+Especially more examples for cmdlets are always welcome.
+
+Head over to the <https://github.com/MicrosoftDocs/Powershell-Docs> repo, and read their [contributing manual](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md)
+
+To do small contributions like typos, just navigate to that file, click the 'Edit' button, and GitHub will create a copy (fork) on your own GitHub for you.
+When you are satisfied with your changes, you can save them, and submit a pull request to the _staging_ branch of the PowerShell-Docs repository.
+
+
+For bigger contributions, you are better off cloning your fork locally to your computer.
+This enables you to use VSCode and other editing tools.
+A guide on how to do that can be found [here](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/1-GET-STARTED.md#making-major-edits-to-existing-topics).
+
+With more examples, less typos, clearer wording, PowerShell can be easier to learn and use for everyone!
+
+### 7. Write Pester tests
+
+> Requires
+> 
+> - GitHub account
+> - Sign [Microsoft Contributor License Agreement](https://cla.opensource.microsoft.com/)
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+> - Know how to run tests
+> - Pester 4.2 or higher
+> - Pester knowledge 
+
+Within the PowerShell project, many parts are tested automatically.
+A big part of that is done with Pester.
+See [here](https://github.com/PowerShell/PowerShell/tree/master/test/powershell).
+
+
+Many CmdLets could use better or more extensive tests to check if they keep working correctly when someone would change them in the future.
+So the Pester tests for CmdLets are kind of a safeguard against regression.
+When there are more/better tests for CmdLets, you can possibly prevent future bugs!
+
+See also "Details about contributing to the PowerShell/PowerShell repo", especially the section on how to run tests.
```

Comment:
I would recommend making that a link. Most Markdown renderers add anchor points on a heading. The ones I've used work kinda like this:

```md
## Heading Name - Thing

some text here

## some other heading

[link](#heading-name-thing)
```

i.e., it uses the heading name minus special characters, and replaces spaces with `-` dashes

Reply:
Yes, great idea! It had crossed my mind, but hadn't looked up the syntax yet :-)

Reply:
fixed now.